src/app.js

```diff
@@ -6,8 +6,193 @@ import _ from 'underscore';
 import './css/foundation.css';
 import './css/style.css';
 
-console.log('it loaded!');
+// models and collections
+import Trip from './app/models/trip';
+// import Reservation from './app/models/reservation';
+import TripList from './app/collections/trip_list';
 
+
+const TRIP_FIELDS = ['name', 'continent', 'category', 'weeks', 'price'];
+const RES_FIELDS = ['name', 'email'];
+
+let allTripsTemplate;
+let showDetailsTemplate;
+let statusMessageTemplate;
+
+// Render Methods
+
+const renderAll = function renderAll(tripList) {
+  const tripListElement = $('#trips-list');
+  tripListElement.empty();
+  $('th.sort').removeClass('current-sort-field');
+  $(`th.sort.${ tripList.comparator }` ).addClass('current-sort-field');
+
+  tripList.on('update', renderAll);
```

Comment:
These should be in `$(document(.ready` this will eventually cause lots of rendering when a new model is added or if it's been sorted multiple times.

Only add the listener once, not every time it renders.